pkgs/tools/inputmethods/netevent/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, docutils }:
+
+stdenv.mkDerivation rec {
+  pname = "netevent";
+  version = "20201018";
+
+  src = fetchFromGitHub {
+    owner = "Blub";
+    repo = "netevent";
+    rev = "ddd330f0dc956a95a111c58ad10546071058e4c1";
```

Comment:
It looks like there are some release tags upstream; Debian has some numbered releases as well; can you use one of those instead of an arbitrary hash? Alternatively maybe you want to file an issue upstream asking for a release

Reply:
The latest release dates from 10 years ago, which is why I'm not using it. I can ask for a release (but the goal is primarily packaging the tool), but in the mean time, i'm using the latest master branch revision.